libs/core/langchain_core/callbacks/base.py

```diff
@@ -179,7 +184,11 @@ def on_chat_model_start(
         metadata: Optional[Dict[str, Any]] = None,
         **kwargs: Any,
     ) -> Any:
-        """Run when a chat model starts running."""
+        """Run when a chat model starts running.
+
+        **ATTENTION**: This method is called for chat models. If you're implementing
+            a handler for a non-chat model, you should use on_llm_start instead.
+        """
         raise NotImplementedError(
```

Comment:
Why do we throw a NotImplementedError here instead of doing a pass. Was this accidentally left here when we made the methods non-required? Or is it to raise awareness that it's different from on_llm_start?